tests/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+const rules = {
+  "@typescript-eslint/no-floating-promises":"off"
```

Comment:
This file is unnecessary to be added.

Fix the exactly problem if there's any.